tests/conftest.py

```diff
@@ -261,43 +271,24 @@ def dummy_carbon_data(fixture_core_components):
 
     # The layer dependant data has to be handled separately - at present all of these
     # are defined only for the topsoil layer
-    data["soil_moisture"] = fixture_core_components.layer_structure.from_template()
-    data["soil_moisture"].loc[
-        {"layers": fixture_core_components.layer_structure.role_indices["topsoil"]}
-    ] = [232.61550125, 196.88733175, 126.065797, 75.63195175]
-
-    data["matric_potential"] = fixture_core_components.layer_structure.from_template()
-    data["matric_potential"].loc[
-        {"layers": fixture_core_components.layer_structure.role_indices["topsoil"]}
-    ] = [-3.0, -10.0, -250.0, -10000.0]
-
-    data["soil_temperature"] = fixture_core_components.layer_structure.from_template()
-    data["soil_temperature"].loc[
-        {"layers": fixture_core_components.layer_structure.role_indices["topsoil"]}
-    ] = [35.0, 37.5, 40.0, 25.0]
-    data["soil_temperature"].loc[
-        {"layers": fixture_core_components.layer_structure.role_indices["subsoil"]}
-    ] = [22.5, 22.5, 22.5, 22.5]
-
-    return data
+    lyr_str = fixture_core_components.layer_structure
```

Comment:
This is a minor thing but could this be `lyr_struct` or something instead? I'm primed to read "str" as "string", so end up spending time wondering what the layer string is used for

Reply:
why not `layer_structure`?

Reply:
Honestly, just to keep the line wrapping sane, but I'll have a look 😄

Reply:
fair enough, that makes a lot of sense